internal/pkg/admission/policy-server-deployment.go

```diff
@@ -345,6 +358,23 @@ func (r *Reconciler) deployment(configMapVersion string, policyServer *policiesv
 	if templateAnnotations == nil {
 		templateAnnotations = make(map[string]string)
 	}
+	if r.MetricsEnabled {
+		envvar := corev1.EnvVar{Name: constants.PolicyServerEnableMetricsEnvVar, Value: "1"}
+		logFmtEnvVar := corev1.EnvVar{Name: constants.PolicyServerLogFmtEnvVar, Value: "otlp"}
+
+		if hasEvnvar, index := envVarsContaintVariable(&admissionContainer, constants.PolicyServerEnableMetricsEnvVar); hasEvnvar {
+			admissionContainer.Env[index] = envvar
+		} else {
+			admissionContainer.Env = append(admissionContainer.Env, envvar)
```

Comment:
I would move the definition of `envvar` inside of this block

Reply:
What do you mean? Duplicate the `envvar` definition on line 366 and 368?

Reply:
Scratch it... I don't know what I saw :facepalm: